src/packages/swipe/swipe.scss

```
@@ -31,7 +31,9 @@
   }
 
   &-right {
+    display: flex;
     //left: 100%;
```

Stylelint (17.10.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)